### ISSUE TITLE
Fix CVP pipeline issues

### ISSUE THREAD

**Issue by phracek** [sclorg/container-common-scripts#221]
During the testing in CVP this issue was caught:

```bash
Testing in CVP environment. No need to login to OpenShift cluster. This is already done by CVP pipeline.
In project cvp-5c79a57dd70cc51dd4c37825-jlgfj (CVP sandbox) (cvp-5c79a57dd70cc51dd4c37825-jlgfj) on server https://172.27.0.1:443

You have no services, deployment configs, or build configs.
Run 'oc new-app' to create an application.
Testing in CVP environment. No need to create OpenShift project. This is done by CVP pipeline
Testing image nodejs-14 in CVP pipeline.
error: only a partial match was found for "nodejs-14:14": "cvp-5c79a57dd70cc51dd4c37825-jlgfj/nodejs-14:latest"

The argument "nodejs-14:14" only partially matched the following Docker image, OpenShift image stream, or template:

* Image stream "nodejs-14" (tag "latest") in project "cvp-5c79a57dd70cc51dd4c37825-jlgfj"
  Use --image-stream="cvp-5c79a57dd70cc51dd4c37825-jlgfj/nodejs-14:latest" to specify this image or template

Waiting for nodejs-14-testing pod becoming ready .................................................................................................... FAIL
NAME                                       IMAGE REPOSITORY                                                                                TAGS      UPDATED
imagestream.image.openshift.io/nodejs-14   image-registry.openshift-image-registry.svc:5000/cvp-5c79a57dd70cc51dd4c37825-jlgfj/nodejs-14   latest    5 minutes ago
In project cvp-5c79a57dd70cc51dd4c37825-jlgfj (CVP sandbox) (cvp-5c79a57dd70cc51dd4c37825-jlgfj) on server https://172.27.0.1:443

You have no services, deployment configs, or build configs.
Run 'oc new-app' to create an application.
Build config bc/ does not exist for some reason.
Import probably failed.
OpenShift tests for nodejs-14 failed.
```
There are two possibilities:
1)  In case of CVP, we have to use tag `latest`.
2) During the import into OpenShift 4, we can tag it to nodejs-14:14.
The taging can be done in file: https://github.com/sclorg/container-common-scripts/blob/master/test-openshift.yaml
especially after this ansible command:
https://github.com/sclorg/container-common-scripts/blob/master/test-openshift.yaml#L53

In my opinion, the first option is better.

WDYT?

**Comment by phracek** [comment]
In case of nodejs-14, import looks like:
```bash
oc import-image nodejs-14 --from=registry.access.redhat.com/ubi7/nodejs-14:latest --insecure=true --confirm
```

**Comment by phracek** [comment]
The changes in container-common-scripts which has to be done are in this functions:

* https://github.com/sclorg/container-common-scripts/blob/master/test-lib-openshift.sh#L689
* https://github.com/sclorg/container-common-scripts/blob/master/test-lib-openshift.sh#L832

The change should be:
```bash
image_tagged="${image_name_no_namespace%:*}:latest"
```

**Comment by phracek** [comment]
Closed in favor of #280
